openapi/components/schemas/Membership.yaml

```
@@ -26,10 +23,7 @@ properties:
     type: object
     properties:
       id:
-        description: Organization ID.
-        type: string
-        allOf:
-          - $ref: ./ResourceId.yaml
+        $ref: ./OrganizationId.yaml
       name:
         description: First and last name.
```

nitpick: I'm assuming this is the Rebilly user's first and last name
```suggestion
        description: Your first and last name.
```